lib/agoneum/web/views/error_helpers.ex

```diff
@@ -9,9 +9,9 @@ defmodule Agoneum.Web.ErrorHelpers do
   Generates tag for inlined form input errors.
   """
   def error_tag(form, field) do
-    if error = form.errors[field] do
+    Enum.map(Keyword.get_values(form.errors, field), fn (error) ->
```

Comment:
This came from [Phoenix](https://github.com/phoenixframework/phoenix/blob/v1.3.0-rc.2/installer/templates/new/web/views/error_helpers.ex#L11-L15)